### PR TITLE
simplify dask.array to numpy route

### LINKED ISSUE #225: dask rec_concatenate doesn't exist anymore

### PR DESCRIPTION
**Pull request by mrocklin**
Fixes #225

### CODE REVIEW

**Comment by cpcloud** [comment]
`discover` also needs to be changed to `from_numpy(a.shape, a.dtype)`

**Comment by mrocklin** [comment]
done

**Comment by cpcloud** [comment]
we're not installing from source with travis, can you remove the `dask` from the conda install line and add a `pip install git://github.com/ContinuumIO/dask.git` in there

**Comment by mrocklin** [comment]
done

**Comment by cpcloud** [comment]
thx

File: .travis.yml [hunk 1]
```diff
@@ -37,6 +37,7 @@ install:
 
   # Install various deps
   - pip install sas7bdat
+  - pip install git://github.com/ContinuumIO/dask.git --upgrade
```
Comment (cpcloud) [inline .travis.yml]: sorry my original line was wrong, it needs a `git+` before the `git://`

**Comment by cpcloud** [comment]
thanks @mrocklin

**Pull request merged**